queries/html_tags/injections.scm

```diff
@@ -0,0 +1,7 @@
+((style_element
+  (raw_text) @css))
+
+((script_element
+  (raw_text) @javascript))
+
+(comment) @comment @combined
```

Comment:
We need to revert the combined from here p:

Reply:
sorry, completely missed that :p